templates/user.tmpl

```diff
@@ -46,7 +46,7 @@
                 {{with .User}}
                     <p>
                     {{if ne .DeletedBy.UserID -1}}
-                        Deleted by {{if and (and (eq (len .DeletedBy.Firstname) 0) (eq (len .DeletedBy.Nickname) 0)) (eq (len .DeletedBy.Lastname) 0)}}
+                        Deleted by {{if checkIfNoNameExists .DeletedBy}}
                             UNKNOWN({{.DeletedBy.UserID}}){{else}}<a href="/internal/user/{{.DeletedBy.UserID}}">{{formatUserName .DeletedBy}}</a>
                         {{end}} at {{.DeletedAt.String}}<br>
```

Comment:
Have you considered a `makeUserLink` or similar helper function with this logic? It's used quite frequently from the looks of it.

Reply:
I'll make a bigger function for that

Reply:
Fixed